README.md

```diff
@@ -16,4 +16,11 @@ With this API you can get events based on _when_, _where_ and _what_, for more d
 ## Wikidata.org - Get info from the Wikipedia project
 Wikidata provides machine readable data from the Wikipedia project, licenced under a Creative Commons "No rights reserved" licence called [CC-0](https://creativecommons.org/share-your-work/public-domain/cc0/).
 
-- [wikidata-org.ipynb](wikidata-org.ipynb) - added by [salgo60](https://github.com/salgo60) along with this [video](https://youtu.be/HrfQioXjGZE)
\ No newline at end of file
+- [wikidata-org.ipynb](wikidata-org.ipynb) - added by [salgo60](https://github.com/salgo60) along with this [video](https://youtu.be/HrfQioXjGZE)
+
+### Wikidata and Machinelearning
+As Wiki is an internatinal project in +300 languages and has a lot of articles, Wikidata is a good start for harvesting data to use in machinelearning
+* Example SPARQL queries
+    * All [Swedish politicians in the Swedish Goverment](http://tinyurl.com/y4jo4lva)
+        * just those with a [twitter account](http://tinyurl.com/y4qb32q9)
+* In may started also the [Lexicographical data project](https://www.wikidata.org/wiki/Wikidata:Lexicographical_data/en) with the goal to "describe precisely all words in all languages" in a machinereadable form
```

Comment:
This file is the top level overview of API examples, and will hopefully contain lots of APIs in a year or two. With this in mind, would you agree that it could make sense to:
- put the query examples inside the notebook you made for Wikidata instead, it seems to me more suitable that that contains more extensive details rather than it being in this overview of examples.
- update the description "Wikidata provides machine readable data..." to be as you desire without adding another header?

Reply:
Ping @salgo60